### ISSUE TITLE
Config.xml breaks the extension

### ISSUE THREAD

**Issue by allanhoumark** [ionic-team/vscode-extension#87]
We are having a app/config.xml file due to some cordova plugin (cordova.plugins.diagnostic)
This file makes the extension never finish the project analyse process.

Does the Ionic extension have a related file with the same name?

Maybe it could be a setting to tell the Ionic extension where this file is located in fx. /extension/ionic/config.xml

**Comment by dtarnawsky** [comment]
Hi @allanhoumark ,

With a bug like this, I would need a copy of your project in order to reproduce the problem. Is that something you can share? If so, can you send a zip via email to damian at ionic.io ?

**Comment by allanhoumark** [comment]
Sorry, I cannot share my code due to company policy. But I think it should be reproducible by adding a config.xml file in app/.


This is the content of our config.xml file

```
<!-- Dette er en cordova config fil, som bruges når plugins installeres -->
<?xml version='1.0' encoding='utf-8'?>
<widget id="com.aarstiderne" version="0.0.0" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
    <preference name="cordova.plugins.diagnostic.modules" value="NOTIFICATIONS" />
</widget>
```

Best regards
Allan Houmark

**Comment by dtarnawsky** [comment]
Thanks @allanhoumark ,

This should be fixed in the next version that comes out.

In the meantime if you add this to your config.xml it will fix the problem:
`    <platform name="ios"></platform>`

**Comment by allanhoumark** [comment]
Hi @dtarnawsky

Thank you very much. Very very fast response and super "support".

/Allan Houmark

**Comment by dtarnawsky** [comment]
This issue was fixed as part of the v1.23.6 release.
